test/acceptance/features/environment.py

```diff
@@ -41,7 +41,7 @@ def before_all(_context):
     elif start_sbo == "remote":
         output, code = cmd.run(
             f"{ctx.cli} get deployment --all-namespaces -o json"
-            + " | jq -rc '.items[] | select(.metadata.name == \"service-binding-operator\").metadata.namespace'")
+            + " | jq -rc '.items[] | select(.metadata.name == \"service-binding-operator-controller-manager\").metadata.namespace'")
```

Comment:
this line should be reverted, our operator deployment should be still called `service-binding-operator`, at least in this PR.

Reply:
I changed this because the deployment name, when installing using `make deploy`, is called `service-binding-operator-controller-manager` (see the output of `./bin/kustomize build config/default`):

```
---
apiVersion: apps/v1
kind: Deployment
metadata:
  labels:
    control-plane: controller-manager
  name: service-binding-operator-controller-manager
  namespace: service-binding-operator-system
spec:
  replicas: 1
  selector:
    matchLabels:
      control-plane: controller-manager
  template:
    metadata:
      labels:
        control-plane: controller-manager
    spec:
      containers:
      - args:
        - --leader-elect
        command:
        - /manager
        image: 192.168.49.2:5000/sbo:991bc81d
```

I got into this command from the following log:

```
$ make deploy                                    \
	   OPERATOR_REPO_REF=$(minikube ip):5000/sbo \
	   SKIP_REGISTRY_LOGIN=true
> > /home/isuttonl/src/service-binding-operator/bin/controller-gen "crd:trivialVersions=true,preserveUnknownFields=false" rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=config/crd/bases
docker build -f Dockerfile -t 192.168.49.2:5000/sbo:991bc81d .
Sending build context to Docker daemon  53.19MB
(snip)
cd config/manager && /home/isuttonl/src/service-binding-operator/bin/kustomize edit set image controller=192.168.49.2:5000/sbo:991bc81d
/home/isuttonl/src/service-binding-operator/bin/kustomize build config/default | kubectl apply -f -
(snip)
```

Then confirmed in the cluster:

```
$ kubectl -n service-binding-operator-system get all
NAME                                                               READY   STATUS              RESTARTS   AGE
pod/service-binding-operator-controller-manager-64dc64c7cb-x59bl   0/1     ContainerCreating   0          22m
pod/service-binding-operator-controller-manager-68fbd6f645-wfwjf   0/1     ContainerCreating   0          3h20m

NAME                                               TYPE        CLUSTER-IP     EXTERNAL-IP   PORT(S)   AGE
service/service-binding-operator-webhook-service   ClusterIP   10.97.49.240   <none>        443/TCP   3h26m

NAME                                                          READY   UP-TO-DATE   AVAILABLE   AGE
deployment.apps/service-binding-operator-controller-manager   0/1     1            0           3h26m
```